packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -212,19 +228,36 @@ export const getComponentCss = (
         })),
         position: 'relative',
         justifyContent: isInfinitePagination ? 'flex-start' : 'center',
-        width: `calc(${paginationActiveBulletSize} + ${paginationBulletSize} * 4 + ${spacingStaticSmall} * 4)`, // Width for five bullets (one active + spacing)
-        left: 'calc(50% - 42px)',
+        width: paginationWidth,
+        left: `calc(50% - (${paginationWidth}) / 2)`,
+        ['@media (pointer: coarse)']: {
```

Comment:
Maybe extract this media query into a const with a speaking name since it appears like 3 times.